browser/search/ntp_utils.cc

```diff
@@ -0,0 +1,48 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/search/ntp_utils.h"
+
+#include "base/command_line.h"
```

Comment:
I think many included headers here can be deleted.

Reply:
fixed